docs/source/qvm.rst

```diff
@@ -33,8 +33,16 @@ The Rigetti Quantum Virtual Machine is an implementation of the Quantum Abstract
 *A Practical Quantum Instruction Set Architecture*. [1]_  It is implemented in ANSI Common LISP and
 executes programs specified in Quil.
 
-The QVM is a wavefunction simulation of unitary evolution with classical control flow
-and shared quantum classical memory.
+The QVM is a wavefunction simulation of unitary evolution with
```

Comment:
Something rubs me the wrong way about this sentence. Alternatively
> The QVM simulates the unitary evolution of a wavefunction with classical control flow.

Reply:
Like that a lot better